source/kubernetes/kubectl-plugin-reference.rst

```diff
@@ -0,0 +1,620 @@
+=================================
+MinIO Kubernetes Plugin Reference
+=================================
+
+.. default-domain:: minio
+
+.. contents:: On This Page
+   :local:
+   :depth: 2
+
+The ``kubectl-minio`` plugin wraps the MinIO Kubernetes Operator (``operator``)
+and provides a simple interface through ``kubectl`` for deploying and managing
+MinIO resources. The ``kubectl-minio`` interface does not require familiarity
+with the Kubernetes API, the MinIO ``operator`` API extension, or the creation
+of ``YAML`` specifications. 
+
+The MinIO ``kubectl-minio`` plugin is a part of the 
+:minio-git:`MinIO Kubernetes Operator <operator/tree/master/kubectl-minio>` 
+Github repository.
+
+Installation
+------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+- Kubernetes v1.17.0 or later
+
+- One Persistant Volumes (``PV``) created using the 
+  :doc:`MinIO Direct CSI </kubernetes/direct-csi>`
+  plugin for each :mc-cmd:`volume <kubectl minio tenant create volumes>` 
+  the ``minio`` server uses. For example, if the MinIO deployment will have
+  4 servers with 4 volumes each, the Kubernetes cluster should have 16
+  total ``PV`` created using the MinIO Direct CSI plugin.
+
+- ``kubectl`` configured for access to the target Kubernetes cluster. Note
+  that ``kubectl`` is suppored within one minor version older or newer than
+  the ``kube-apiserver`` on the destination cluser. See
+  the Kubernetes documentation 
+  :kube-docs:`Kubernetes version and version skew support policy
+  <setup/release/version-skew-policy>`
```

Comment:
We can skip 
```
Note that ``kubectl`` is suppored within one minor version older or newer than
the ``kube-apiserver`` on the destination cluser. See
the Kubernetes documentation 
  :kube-docs:`Kubernetes version and version skew support policy
  <setup/release/version-skew-policy>`
```
IMHO a working `kubectl` and its compatibility with the kubernetes cluster is our of scope for us.